GliaWidgets/Sources/View/Chat/Message/Content/Text/ChatTextContentView.swift

```diff
@@ -110,7 +109,8 @@ class ChatTextContentView: BaseView {
         textView.accessibilityIdentifier = text
     }
 
-    private func setAttributedText(_ text: NSAttributedString?) {
+    private func setAttributedText(_ text: NSMutableAttributedString?) {
+        print(textView.attributedText)
```

Comment:
Do you need it?

Reply:
fixed [e84326b](https://github.com/salemove/ios-sdk-widgets/pull/705/commits/e84326b7cd81b80cecc5ec5b13b6881ff6057432)